controllers/cloud.redhat.com/version.txt

```diff
@@ -1 +1 @@
-master
\ No newline at end of file
+v0.30.0-26-g1b4289d
```

Comment:
This file should not be committed/updated `make pre-push` could help here

Reply:
I promise that at some point in the next few years I'll get this